Shared/Samples/Apply class breaks renderer to sublayer/ApplyClassBreaksRendererToSublayerView.swift

```diff
@@ -0,0 +1,123 @@
+// Copyright 2025 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct ApplyClassBreaksRendererToSublayerView: View {
+    /// The map image layer.
+    private let mapImageLayer = ArcGISMapImageLayer(url: URL(string: "https://sampleserver6.arcgisonline.com/arcgis/rest/services/Census/MapServer")!)
+    
+    /// The counties sublayer.
+    @State private var countiesLayer: ArcGISMapImageSublayer?
+    
+    /// The original renderer used to symbolize the sublayer.
+    @State private var originalRenderer: Renderer?
+    
+    /// A Boolean value indicating that the class breaks renderer is applied.
+    @State private var applyClassBreaksRenderer = false
+    
+    /// The map with a topographic basemap style.
+    @State private var map: Map = {
+        let map = Map(basemapStyle: .arcGISTopographic)
+        
+        // Sets the initial viewpoint.
+        map.initialViewpoint = Viewpoint(
+            boundingGeometry: Envelope(
+                xMin: -13934661.666904,
+                yMin: 331181.323482,
+                xMax: -7355704.998713,
+                yMax: 9118038.075882,
+                spatialReference: .webMercator
+            )
+        )
+        return map
+    }()
+
+    /// The error shown in the error alert.
+    @State private var error: Error?
+
+    var body: some View {
+        MapView(map: map)
+            .task {
+                // Adds the map image layer to the map.
+                map.addOperationalLayer(mapImageLayer)
+                await loadMapImageSublayer()
+            }
+            .toolbar {
+                ToolbarItem(placement: .bottomBar) {
+                    Toggle("Change Sublayer Renderer", isOn: $applyClassBreaksRenderer)
+                }
+            }
+            .onChange(of: applyClassBreaksRenderer) {
+                countiesLayer?.renderer = if applyClassBreaksRenderer {
+                    // Applies the class breaks renderer.
+                    .populationRenderer
+                } else {
+                    // Applies the original renderer.
+                    originalRenderer
+                }
+            }
+            .errorAlert(presentingError: $error)
+    }
+    
+    func loadMapImageSublayer() async {
```

Comment:
This function only loads the map image sublayer so that it can grab the counties layer and the original renderer. In that regard I think the name doesn't convey the purpose of the function.

Reply:
Rather than rename the function, I moved the code into the task. I found that we have several samples that do something similar.